src/codegen.cpp

```diff
@@ -519,14 +518,12 @@ typedef struct {
     bool vaStack;      // varargs stack-allocated
     bool sret;
     int nReqArgs;
-    std::vector<bool> boundsCheck;
-    std::vector<bool> inbounds;
 
     CallInst *ptlsStates;
     Value *signalPage;
 
-    llvm::DIBuilder *dbuilder;
     bool debug_enabled;
+    bool is_inbounds{false};
```

Comment:
Note that this is also what https://github.com/JuliaLang/julia/pull/18125/files does https://github.com/JuliaLang/julia/blob/b7f3127940df7bf779d377865aee04490a5a6e0a/src/codegen.cpp#L830